src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -121,6 +121,24 @@ public boolean clickEditInstructorLink(int instrNum) {
         return isEditable;
     }
     
+    public void clickCancelEditInstructorLink(int instrNum) {
+        getCancelEditInstructorLink(instrNum).click();
+    }
+    
+    public boolean isInstructorFormNotEditable(int instrNum) {
+        WebElement saveButton = getSaveInstructorButton(instrNum);
+        waitForElementToDisappear(By.id("btnSaveInstructor" + instrNum));
```

Comment:
Perhaps I could omit checking whether the save button is displayed on line 137? Since there is already a wait for it to disappear. I will then be able to omit `WebElement saveButton = getSaveInstructorButton(instrNum);`